adapters/blip_2/dataloop.json

```diff
@@ -0,0 +1,116 @@
+{
+    "name": "blip-2-huggingface-app",
+    "description": "BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models",
+    "attributes": {
+      "Provider": "Salesforce",
+      "Deployed By": "Dataloop",
+      "License": "BSD-3-Clause",
+      "Category": "Model",
+      "NLP": "Conversational",
+      "Gen AI": "LMM",
+      "Media Type": ["Text", "Image", "Multi Modal"]
+    },
+    "displayName": "BLIP 2",
+    "version": "0.1.45",
+    "scope": "public",
+    "codebase": {
+      "type": "git",
+      "gitUrl": "https://github.com/dataloop-ai-apps/huggingface-adapter.git",
+      "gitTag": "0.1.45"
+    },
+    "components": {
+      "computeConfigs": [
+        {
+          "name": "blip-2-huggingface-deploy",
+          "secrets": [],
+          "runtime": {
+            "podType": "regular-l",
```

Comment:
is this the smallest size that worked?

Reply:
Yup, any smaller pod type would result in a init loop :(